Bottom: 915e0ee4505ffa5dc789b46752172eee702adae3
Top:    52637b17efade61815ce9a543e6ac319af39e10a
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-10-25 00:12:16 -0700

Refresh of rs-iomap

---

diff --git a/man/rsocket.7 b/man/rsocket.7
index 5ecebc2..dc3f9ab 100644
--- a/man/rsocket.7
+++ b/man/rsocket.7
@@ -79,7 +79,7 @@ a receiver's view, riowrite transfers are silently redirected into a pre-
 determined data buffer.  Data is received automatically, and the receiver
 is not informed of the transfer.  However, iowrite data is still considered
 part of the data stream, such that iowrite data will be written before a
-subsequent transfer is received.  A  message sent immediately after initiating
+subsequent transfer is received.  A message sent immediately after initiating
 an iowrite may be used to notify the receiver of the iowrite.
 .P
 In addition to standard socket options, rsockets supports options
